server/src/jobs/oneshot/psFormation/bulkUpdate.js

```diff
@@ -0,0 +1,68 @@
+/**
+  * TARGET :
+  * SI le libellé_uai_composante OU libelle_uai_affilié contient "agricole" ou "agriculture" 
+  * ET qu'un établissement matché CONTIENT "CFAA" ou "CFA AGRICULTURE", 
+  * ALORS c'est le formateur de cette formation.
+  * 
+    SI le libellé_uai_composante OU libelle_uai_affilié contient "agricole" ou "agriculture" 
+    ET qu'un établissement matché CONTIENT "LEGTA" ou "ETABLISSEMENT PUBLIC LOCAL D'ENSEIGNEMENT ET DE FORMATION PROFESSIONNELLE AGRICOLE", 
+    ALORS c'est le gestionnaire de cette formation.
+  */
+
+const { asyncForEach } = require("../../../common/utils/asyncUtils");
+const _ = require("lodash");
```

Comment:
Pour le tree shaking :
```suggestion
const uniqBy = require("lodash/uniqBy");
```